stock_location_restrict_group/models/stock_location.py

```diff
@@ -26,7 +26,6 @@ def _compute_restricted_group(self):
         for location in self.filtered(lambda l: l.group_restricted):
             quants = self.env['stock.quant'].search(
                 [('location_id', '=', location.id)])
-            location.restricted_group = False
             rest_group = quants.mapped('history_ids').filtered(
```

Comment:
mmm not exactly, I mean that once we have a restricted_group for a location, that restricted_group field doesn't need to be computed anymore. but I forgot that the computed fields are not stored by default  :sweat_smile: so maybe the field should be <code>store=True</code> and the line <code>if not location.restricted_group:</code> should be added in order to prevent it to change. Or there is any scenario where is ok that the restricted_group for a location changes?